docker/ros-bare.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+set -o errexit
+set -o verbose
+
+echo "deb http://packages.ros.org/ros/ubuntu stretch main" > /etc/apt/sources.list.d/ros-latest.list && 
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+apt update
+apt upgrade 
+apt-get install build-essential python-yaml cmake subversion wget python-setuptools mercurial git-core libapr1-dev libaprutil1-dev libbz2-dev python-dev libgtest-dev python-paramiko libboost-all-dev liblog4cxx10-dev pkg-config python-empy swig python-nose lsb-release python-pip python-gtk2
```

Comment:
These dependencies are needed to compile ROS but do not depend on ROS. So these should be installed in the Dockerfile.

Maybe we could move Dockerfile install dependencies to a file, but I am not sure how Dockerhub would behave.

But we could also move the Docker build to travisCI then push the image to dockerhub

Reply:
Yes but that's not a big deal. It's just gonna increase a little bit the size of the image.
And yes it's gonna make ROS build faster and move us a bit away from the 50 mins timeout